spec/models/blob_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Blob, type: :model do
+  describe "validations" do
+    it "is invalid without a document_id" do
+      blob = Blob.new(document_filename: "test.json", data: "some data")
+      expect(blob).to_not be_valid
+    end
```

Comment:
I probably would have written all of these as one spec though _if_ I were to put presence specs in at all.  I probably would have done something like

```
required_attributes = [:document_id, :document_filename, :data].each do |attr|
  it { is_expected.to validate_presence_of(attr) }
end
```

not sure of the exact syntax, but I _think_ that's close.  Keep it simple and easy to read, but again, I'm not sure that I would have even created a spec for checking presence.

Reply:
I refactor it according suggestion